plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/CitrixResourceBase.java

```diff
@@ -4251,6 +4257,34 @@ public void rebootVM(final Connection conn, final VM vm, final String vmName) th
         }
     }
 
+    protected String skipOrRemoveSR(Connection conn, SR sr) {
```

Comment:
@GabrielBrascher Pointing to oracle is not a good support for your point.

You give a good example of why a well named method should still be documented. It's name does not make clear how it works. This should be addressed in a different way in my opinion: Can this be renamed to better depict what it does and only if all else fails should the user have to RTFM. @rodrigo93 starts with asking for javadoc. Such questions have been asked a lot lately and mostly unjust (not just by @rodrigo93 ) In my opinion this is not the way to improve our codebase. The question could we rename the method (or other code element) to something more descriptive had not been asked.